docs/types/react-docgen.d.ts

```diff
@@ -37,7 +37,7 @@ declare module 'react-docgen' {
   }
   export interface ReactDocgenApi {
     description: string;
-    props: Record<string, PropDescriptor>;
+    props?: Record<string, PropDescriptor>;
```

Comment:
This key should be optional. If the component doesn't support props it's `undefined`.

https://github.com/reactjs/react-docgen/blob/7298b8628e4bd436cfd094f77404fde875ba6c96/src/Documentation.js#L13